spk/rdiff-backup/Makefile

```diff
@@ -21,11 +21,6 @@ LICENSE  = GPLv2
 
 SERVICE_SETUP = src/service-setup.sh
 
-include ../../mk/spksrc.common.mk
-ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
 SPK_USR_LOCAL_LINKS = bin:env/bin/rdiff-backup bin:env/bin/rdiff-backup-statistics
```

Comment:
Just curious, What makes you pick between `SPK_COMMANDS` vs `SPK_USR_LOCAL_LINKS = bin:` ? 

FYI when `ctl_stop` is absent or "Yes" the user can link and unlink the `/usr/local/*` stuff on package start/stop.

Reply:
with `SPK_COMMANDS` the folder is reused unter `/usr/local` i.e. `bin/file` will create `/usr/local/bin/file => /var/packages/{package}/target/bin/file` 

As `SPK_COMMANDS = env/bin/test` will create `/usr/local/bin/env/test => /var/packages/{package}/target/env/bin/test`
this is not expected, 
but with `SPK_USR_LOCAL_LINKS = bin:bin/env/test` you get
`usr/local/bin/test => /var/packages/{package}/target/env/bin/test`

SPK_COMMANDS = `sbin/test` will create a link
`/usr/local/sbin/test => /var/packages/{package}/target/sbin/test` (but I'm not shure whether this is in DSM7 the same).

Reply:
Finally I must confess that we could remove the `SPK_COMMANDS` (but must adjust all packages that currently use it) with a little supplementary setting:
`SPK_COMMANDS = bin/file`
may be replaced by
`SPK_USR_LOCAL_LINKS = bin:bin/file`

Reply:
Or may be we can enhance `SPK_USR_LOCAL_LINKS` to behave like `SPK_COMMANDS` when no colon separater is used.

Reply:
Thanks for the info, yea AFAIK `sbin` is not supported with DSM7. 
We can add that to our long todo list for now since it's not critical and more of a cosmetic change.